test/unit/http/common/backoff_test.dart

```diff
@@ -0,0 +1,179 @@
+// Copyright 2015 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@TestOn('browser')
+import 'dart:async';
+import 'dart:html';
+
+import 'package:test/test.dart';
+
+import 'package:w_transport/src/http/auto_retry.dart';
+
+import '../../../naming.dart';
+import 'package:w_transport/src/http/requests.dart';
+import 'package:w_transport/src/http/common/backoff.dart';
+import 'package:w_transport/w_transport_mock.dart';
+import 'dart:math';
+
+void main() {
```

Comment:
One of my comments got buried with the last commit, so I'll repost:

You'll have to import this file and execute this main block in the platform independent test suite: https://github.com/Workiva/w_transport/blob/master/test/unit/platform_independent_unit_test.dart